core/controlplane/config/templates/cluster.yaml

```diff
@@ -174,8 +174,29 @@ worker:
 #          # Must be within the range between 100 and 2000
 #          # Defaults to worker.spotFleet.unitRootVolumeIOPS * weightedCapacity if omitted
 #          #rootVolumeIOPS:
+#
+#        #  # Additional EBS volumes mounted on the worker
+#        #  # No additional EBS volume by default. All parameter values do not default - they must be explicitly defined
+#        #  ebsVolumes:
+#        #  - type: "gp2"
+#        #    iops: 0
+#        #    sizeGb: 30
+#        #    # Follow the aws convention of '/dev/xvd*' where '*' is a single letter from 'f' to 'z'
+#        #    device: "/dev/xvdf"
+#        #    mountPath: "/ebs"
```

Comment:
If we go ahead with https://github.com/coreos/kube-aws/pull/342/files#r102595612, `mountPath` should be just `path` because the `mount` part would be redundant in `worker.volumeMounts[].mountPath`